corehq/apps/users/views/utils.py

```diff
@@ -113,26 +114,46 @@ def log_commcare_user_locations_changes(request, user, old_location_id, old_assi
         )
 
 
-def get_locations_with_cases(domain, location_ids):
-    """
-    Takes a list of location IDs and returns how many cases are assigned to each location
-    :param domain: The domain to search in
-    :param location_ids: A list of location IDs to get case counts on
-    :returns A dict with location names as the key, and the number of cases assigned to it as the value
-    """
+def _get_location_ids_with_single_user(domain, location_ids, user_id):
+    other_commcare_user_ids = [user.user_id for user in get_all_commcare_users_by_domain(domain)]
```

Comment:
Why not use `get_all_user_ids_by_domain` instead?

Reply:
I have changed the implementation to get the user ids from UserES, as per your suggestion.